Tools/Gulp/config.json

```diff
@@ -1170,6 +1171,16 @@
                 "additionalTextures"
             ]
         },
+        "particleHelper": {
+            "files": [
+                "../../src/Helpers/babylon.particleHelper.js"
+            ],
+            "dependUpon": [
+                "core",
+                "backgroundMaterial",
```

Comment:
you should just rely on particles actually

Reply:
yeah that's what I said I was not sure at all what to do here so just : 
```json
"dependUpon": [
    "particles"
]
```

is good?